tests/OnlineSales.Tests/SimpleTableTests.cs

```diff
@@ -48,17 +50,39 @@ public virtual async Task UpdateItemNotFoundTest()
     }
 
     [Fact]
-    public async Task CreateAndUpdateItemTest()
+    public virtual async Task CreateAndCheckEntityState_ChangeLog()
     {
         var testCreateItem = await CreateItem();
 
-        var testUpdateItem = UpdateItem(testCreateItem.Item1);
+        var item = await GetTest<T>(testCreateItem.Item2);
 
-        await PatchTest(testCreateItem.Item2, testUpdateItem!);
+        var result = Get<ChangeLog>(c => c.ObjectId == item!.Id && c.ObjectType == typeof(T).Name && c.EntityState == EntityState.Added);
 
-        var item = await GetTest<T>(testCreateItem.Item2);
+        result.Should().NotBeNull();
+    }
 
-        item.Should().BeEquivalentTo(testCreateItem.Item1);
+    [Fact]
+    public async Task CreateAndUpdateItemTest()
+    {
+        var createAndUpdateItems = await CreateAndUpdateItem();
+
+        var item = await GetTest<T>(createAndUpdateItems.testCreateItem.Item2);
+
+        item.Should().BeEquivalentTo(createAndUpdateItems.testCreateItem.Item1);
+    }
+
+    [Fact]
+    public virtual async Task CreateAndUpdateCheckEntityState_ChangeLog()
+    {
+        var createAndUpdateItems = await CreateAndUpdateItem();
+
+        var item = await GetTest<T>(createAndUpdateItems.testCreateItem.Item2);
+
+        var result = Get<ChangeLog>(c => c.ObjectId == item!.Id && c.ObjectType == typeof(T).Name && c.EntityState == EntityState.Modified);
+
+        result.Should().NotBeNull();
+
+        result!.EntityState.Should().Equals(EntityState.Added);
```

Comment:
This test will never be able to pass as you find records with EntityState.Modified status and here you check it for EntityState.Added. Am I missing something?